lib/Maui.GoogleMaps.Clustering/Platforms/Android/ClusterLogic.cs

```diff
@@ -180,6 +180,7 @@ private void AddMarker(Pin outerItem, ClusteredMarker marker)
         {
             outerItem.NativeObject = marker;
             onMarkerCreated(outerItem, marker);
+            if(outerItem.Label != "Pin 5")
```

Comment:
wtf is this?)

Reply:
It was a test that ended up staying in the project and was removed.